include/drivers/dma.h

```diff
@@ -42,6 +42,12 @@ enum dma_addr_adj {
 	DMA_ADDR_ADJ_NO_CHANGE,
 };
 
+/* channel attribute */
```

Comment:
Please, add a comment explaining what `NORMAL` and `PERIODIC` mean.

Reply:
ok.